README.md

```diff
@@ -1,3 +1,30 @@
 # niftimath-rs
 
-NiftiMath rust project.
\ No newline at end of file
+NiftiMath rust project.
+
+This project provide a simple way to make mathematical operation on nifti images (the 
+same way mrcalc from [mrtrix3](https://github.com/MRtrix3/mrtrix3)).
```

Comment:
provideS. I'm pretty sure it's "to do", not "to make". Also, the ending of your sentence is not grammatically correct. maybe "in the same way as mrcalc" or "as mrcalc does", etc.